### PR TITLE
feat(enforce-consistent-line-wrapping): add printWidth fixer

### PR DESCRIPTION
**Pull request by dawnmist**
Issue: enable migrating from using the printWidth fixers used in eslint-plugin-better-tailwindcss

Where there are components that have classes such as:
```
      flex min-w-0 flex-1 flex-col items-center justify-center gap-6 rounded-lg border-dashed p-6
      text-center text-balance
      md:p-12
```
and others with classes like:
```
      has-[[data-slot=input-group-control]:disabled]:opacity-50
      has-[[data-slot=input-group-control]:focus-visible]:ring-4
      has-[[data-slot=input-group-control]:focus-visible]:ring-ring
```
using a strict `classesPerLine` format is problematic - in some cases joining two classes on a single line can create a very long line to read/scroll horizontally, while restricting other lines to only two classes produces a lot of very short lines requiring a lot of vertical scrolling to see the full class set.

Using the `printWidth` option for the `enforce-consistent-line-wrapping` rule will currently produce an error when the `printWidth` is exceeded, but doesn't apply a fix for that error.

Using oxfmt's `sortTailwindcss` results in many class lines being merged, creating lines at 300+ characters long even with the `preserveWhitespace` option enabled. It's also not possible to limit the tailwindcss format to a specific section of a project - working in a legacy monolith that is updating to using tailwindcss means that oxfmt treats all css as tailwind instead of the dedicated section of the project where tailwind is actually used.

Running `eslint-plugin-better-tailwindcss` within oxlint as a jsPlugin adds significant time to the oxlint linting process (its addition for us was about 10 times longer than running oxlint-tailwindcss adds - taking oxlint from 3 seconds to 13 seconds just by adding the better-tailwindcss plugin). That time addition defeats the purpose of migrating away from eslint to oxlint.

This PR adds a fixer to the `printWidth` rule option based on the logic used in `eslint-plugin-better-tailwindcss`'s default fixer for this rule.

New options:
- `wrapLines`, options `overWidth` or `all`: enables the width fixer, and specifies whether to wrap only those lines inside a template that exceed the printWidth or all lines in a template. If not specified, the rule remains in reporting only status (no fixes applied).
- `group`, options `never`, `newLine`, or `emptyLine`, default `newLine`: when `wrapLines` is set to `all`, specifies how variant groups should wrap, mirroring the same `group` option/behaviour and default setting from `eslint-plugin-better-tailwindcss`.

The fixer is only enabled when `wrapLines` is set and `classesPerLine` is not set.

When `wrapLines` is set to `overWidth`, only lines that exceed the printWidth are rewrapped. These are wrapped to fit the printWidth in a greedy algorithm without reordering classes and without grouping by variants.

Changed behaviour:
- Suppress logging errors/warnings if a line containing a single class exceeds the printWidth. Given that there is a single class it is not possible to act on that warning/error, making it only noise.
- Rule is now design-system optional. When using `wrapLines: all` with group set to `newLine` or `emptyLine`, the design-system is consulted for the tailwindcss prefix so that this prefix can be ignored/removed from the classnames when checking for variant groups. If the `entryPoint` is not set, the fixer falls back to treating a prefix as part of the variant group.

PR was prepared using Claude, and tested against our project. I unfortunately do not read/speak Spanish, so I can't check that part of Claude's work - if it needs changing I'm happy to adjust it.

### CODE REVIEW

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 1]
```diff
@@ -119,19 +201,46 @@ export const enforceConsistentLineWrapping = defineRule({
...
-          })
+        const data = {
+          length: String(maxLine),
+          printWidth: String(printWidth),
+        }
+        const isMultiline = loc.value.includes('\n')
+        if (
+          classesPerLine === undefined &&
+          loc.node.type === 'TemplateElement' &&
+          (maxLine > printWidth || isMultiline)
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: **This `|| isMultiline` is the line that changes the rule's default behavior, and I think it's too aggressive to ship on by default.**

`printWidth` defaults to 80 (`defaultOptions` above) and `classesPerLine` is undefined by default, so this branch is live with zero config. And because it fires on `isMultiline` alone, *any* multiline template gets pulled in, even one whose longest line is 20 chars. So someone who already has this rule on (no options, or just `printWidth`) suddenly gets every multiline `className` reformatted into the variant-grouped layout plus a new `inconsistentWrapping` report, where before `printWidth` was warn-only and only fired when a line actually passed 80.

Repro, no options at all:
```tsx
const className = `
  flex
  items-center
  p-4
`
```
Every line fits well under 80. On `main` this is untouched. With this branch it reports `inconsistentWrapping` and rewrites to `flex items-center p-4` on one line.

This is the piece that cuts against the v1.0.0 philosophy. My ask: make the reformatting opt-in. A `group`/`wrap` option, default off, keeps `printWidth` warn-only as the default and lets teams that want the better-tailwindcss layout turn it on. Same capability, no surprise for existing configs. The grouping helpers below are where it would plug in.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 2]
```diff
@@ -111,6 +162,37 @@ export const enforceConsistentLineWrapping = defineRule({
...
+     * fix never converges.
+     */
+    function rewrapTemplateToWidth(loc: ClassLocation, printWidth: number): string {
+      const baseIndent = deriveBaseIndent(loc)
+      const interiorIndent = baseIndent + INDENT_UNIT
+      const leadingReserve = loc.preserveLeadingSpace ? 1 : 0
+      const trailingReserve = loc.preserveTrailingSpace ? 1 : 0
+      const classes = splitClasses(loc.value)
+      if (classes.length === 0) return loc.value
+      if (!loc.preserveLeadingSpace && !loc.preserveTrailingSpace) {
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: **This gate is the one real correctness bug I found, and it fires under the default config on `${}`-in-`className`, about as common a pattern as there is.**

The gate treats any quasi adjacent to a `${}` as a hanging fragment. That's right for a quasi that *follows* an interpolation (`preserveLeadingSpace`), but wrong for one that only *precedes* it (`preserveTrailingSpace` only), i.e. the text right after the opening backtick. That leading quasi falls into the hanging-join branch below (no leading newline), so its packed content gets glued onto the physical line that already holds `const className = \``. And since the `maxLine` loop measures each quasi in isolation and never the absolute source column, the rule can't see the over-width result and never re-flags it. The fix converges to a state that violates the very `printWidth` it's enforcing.

Repro (`printWidth: 40`):
```tsx
const className = `
  flex items-center justify-between gap-4 rounded-lg
  ${isActive ? "x" : "y"}
  px-6 py-3 shadow-md
`
```
becomes
```tsx
const className = `flex items-center justify-between
  gap-4 rounded-lg ${isActive ? "x" : "y"} px-6 py-3 shadow-md`
```
Line 1 is 52 columns, line 2 is 63, both over 40, and it re-lints clean. A simpler one under the *default* `printWidth` 80: `` `\n  flex items-center\n  ${cond}\n  p-4 m-2\n` `` collapses onto a single line.

Fix looks like keying the standalone-vs-hanging decision off `preserveLeadingSpace` only, so a leading (`preserveTrailingSpace`-only) quasi takes the block form.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 3]
```diff
@@ -22,6 +23,54 @@ function chunkList(classes: string[], n: number): string[] {
...
+  if (current !== '') lines.push(current)
+  return lines
+}
+
+/**
+ * Split `classes` into runs of consecutive classes sharing the same variant
+ * chain ("hover:", "md:hover:", "" for base utilities), so wrapping breaks
+ * between variant groups rather than mid-group.
+ */
+function groupByVariantRun(classes: string[]): string[][] {
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: The variant-run grouping itself is nicely built. Two things about how it's wired, though:

**1. It's hardcoded as the only accepted layout.** The schema exposes just `printWidth` and `classesPerLine`, so there's no way to turn the grouping off. In better-tailwindcss the equivalent (`group: newLine`) is a configurable option, and I'd want the same here. This is where the opt-in flag from the `isMultiline` comment would live.

**2. Because of that, hand-formatted layouts get reflown.** A team that wraps by concern instead of by variant has it regrouped, and runs that fit get collapsed onto one line. Reproduced with no options: `` `\n  flex items-center\n  justify-between p-4\n` `` (readable 2-line, all under 80) collapses to a single interior line.

Neither is a correctness problem, the grouping is deterministic and converges. It's a product call, and I'd rather ship the grouping as opt-in than make one wrapping opinion mandatory by default.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 4]
```diff
@@ -119,19 +201,46 @@ export const enforceConsistentLineWrapping = defineRule({
...
+          (maxLine > printWidth || isMultiline)
+        ) {
+          // printWidth set without classesPerLine: enforce (and autofix to)
+          // variant-grouped lines packed to the print width. A multiline
+          // template that already fits is still normalized to that canonical
+          // form; one that matches it is valid. Compare AFTER preserveSpaces:
+          // a fragment's raw value carries the preserved space, and comparing
+          // without it re-reports an already-canonical fragment with a no-op
+          // fix.
+          const fixedValue = preserveSpaces(loc, rewrapTemplateToWidth(loc, printWidth))
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: Heads up on a semantics bug this path can trigger. `preserveSpaces` prepends a space for every non-first quasi (`preserveLeadingSpace` is set to `i > 0` in `extractors.ts` without checking whether a real space was actually there). When the width branch fixes a glued fragment, `${a}flex` becomes `${a} flex`, which splits one runtime class into two.

Repro (`printWidth: 40`): `` `${a}flex items-center gap-2 p-4 m-2 bg-white text-black rounded-lg shadow` `` fixes to `` `${a} flex items-center gap-2 p-4 m-2\n  bg-white text-black rounded-lg shadow` `` (note the injected space after `${a}`).

It's pre-existing in the `classesPerLine` path, but this branch is the first autofix reachable under a `printWidth`-only config, so it's new exposure. Rare form, but it's a silent class corruption on `--fix`, so worth guarding (only prepend the space when the raw quasi actually had one).

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 5]
```diff
@@ -111,6 +162,37 @@ export const enforceConsistentLineWrapping = defineRule({
...
+      const leadingReserve = loc.preserveLeadingSpace ? 1 : 0
+      const trailingReserve = loc.preserveTrailingSpace ? 1 : 0
+      const classes = splitClasses(loc.value)
+      if (classes.length === 0) return loc.value
+      if (!loc.preserveLeadingSpace && !loc.preserveTrailingSpace) {
+        const packed = packGroupedToWidth(classes, interiorIndent, printWidth)
+        return '\n' + packed.map((c) => interiorIndent + c).join('\n') + '\n' + baseIndent
+      }
+      // Fragment adjacent to a `${}`: hanging join, no leading/trailing newline.
+      return packGroupedToWidth(
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: Two smaller things about the fragment branch, both only for templates with interpolations:

**1. Staircase indent with more than one `${}`.** Each quasi is fixed as its own `ClassLocation` and `deriveBaseIndent` reads its current source line. Under iterated `--fix`, quasi[0]'s inserted newlines push quasi[1] onto an indented continuation line, so quasi[1] wraps at 4 spaces while quasi[0] wrapped at 2. It converges and it's whitespace-only, but the output is visibly misaligned, and the fragment tests only cover single-interpolation templates so it isn't caught.

**2. The hanging join ignores the `${expr}` width.** It always glues quasi[k+1]'s first run onto quasi[k]'s tail right after the interpolation, and the packer can't know the expr's width, so the combined physical line can sit well over `printWidth` while both quasis individually "fit", with no diagnostic. Partly inherent to wrapping across an interpolation, but starting quasi[k+1] on a fresh line instead of hanging would bound it.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 6]
```diff
@@ -119,19 +201,46 @@ export const enforceConsistentLineWrapping = defineRule({
         // #110: measure the LONGEST INDIVIDUAL LINE, not the raw total. The raw
         // value of a multiline template includes every `\n` and indent, so
         // comparing its total length made wrapping impossible to satisfy.
+        // A line holding a single class can never be wrapped shorter, so an
+        // over-budget single class is not counted against the print width.
         let maxLine = 0
         for (const line of loc.value.split('\n')) {
-          if (line.length > maxLine) maxLine = line.length
+          if (line.length <= maxLine) continue
+          if (line.length > printWidth && splitClasses(line).length <= 1) continue
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts:365]: Just flagging this as an intentional-looking behavior change to document, not a bug. On `main` the `maxLine` loop counted every line, so a single class longer than `printWidth` reported `tooLong`. Now it's skipped (a lone class can't be wrapped shorter, which is fair). I agree with the change, but it's silent right now, so it's worth a CHANGELOG line since some users will stop seeing a report they used to get.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 7]
```diff
@@ -22,6 +23,54 @@ function chunkList(classes: string[], n: number): string[] {
...
+ * Greedily pack `classes` into space-joined lines, adding classes to the
+ * current line until `indent` + the line would exceed `width`. A single class
+ * that alone exceeds the budget still gets its own line.
+ */
+function packToWidth(classes: string[], indent: string, width: number): string[] {
+  const lines: string[] = []
+  let current = ''
+  for (const cls of classes) {
+    const candidate = current === '' ? cls : current + ' ' + cls
+    if (current !== '' && indent.length + candidate.length > width) {
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts:83]: nit: `indent.length` counts a tab as 1 column, so in tab-indented files the packer under-counts the visual width and can pack a line that renders past `printWidth`. Detection uses the same char-based yardstick, so it still converges (no cycle). Only matters for tabs and it's low priority, maybe a comment that the width budget assumes space indentation.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 8]
```diff
@@ -119,19 +201,46 @@ export const enforceConsistentLineWrapping = defineRule({
...
+          if (fixedValue !== loc.value) {
+            context.report({
+              node: loc.node,
+              messageId: maxLine > printWidth ? 'tooLong' : 'inconsistentWrapping',
+              data,
+              fix(fixer) {
+                return fixer.replaceTextRange(loc.range, fixedValue)
+              },
+            })
+          } else if (maxLine > printWidth) {
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: nit: this `else if (maxLine > printWidth)` is unreachable. It only runs when `fixedValue === loc.value`, i.e. a fixed point of the packer, and at a fixed point every multi-class line is within budget and single-class over-budget lines are already skipped up in the `maxLine` loop, so `maxLine <= printWidth` always holds here. Harmless, but it reads like a canonical layout could still be `tooLong`, which can't happen. Safe to drop.

**Review by sergioazoc — COMMENTED** [review]
Thanks for this, Janeene, and for doing it so thoroughly. The motivation lands: `printWidth` without a fixer is half a feature, and I've hit the exact oxfmt `sortTailwindcss` problem you describe (everything merged onto one 300+ char line, no way to scope it to the part of the repo that actually uses Tailwind). I want this capability in.

I pulled the branch, ran the full suite (green), then spent a while probing the fixer against real inputs. Good news first: the #14 whitespace-vs-wrapping cycle does **not** come back. I fixpoint-tested the width fixer together with `no-unnecessary-whitespace`, `enforce-sort-order` and the count-changing fixers, zero oscillations, and the class order is provably preserved (the grouping only inserts breaks, it never reorders). That was my biggest fear here and it's clean. The Spanish docs read correctly too (neutral tuteo, no voseo). Solid work.

So the feature is a yes. What's holding the PR back is delivery, not the idea. Three things before it can land, then a handful of smaller ones I left inline.

**1. It reformats every multiline template by default.** `printWidth` defaults to 80 and the new branch fires on `|| isMultiline`, so enabling the rule with no options now rewrites every multiline `className` into the variant-grouped layout, where before `printWidth` was warn-only and only fired when a line actually blew past 80. That's a behavior change for existing configs, and it cuts against the v1.0.0 line (deterministic, explicit, the reason auto-detection got removed in the first place). My ask is to make the reformatting opt-in: a `group`/`wrap` option, default off, so `printWidth` stays warn-only unless you turn the layout on. Same feature, no surprise for people already using the rule. Details on the `isMultiline` line and on `groupByVariantRun`.

**2. One real correctness bug.** A template whose first quasi precedes a `${}` gets collapsed onto the backtick line, producing lines longer than `printWidth` that the rule then can't re-detect, so `--fix` rewrites valid code into worse code and calls it done. It fires under the default config on the `${}`-in-`className` pattern, which is everywhere. Full repro on the standalone-gate line.

**3. Release mechanics.** Version is still `1.10.0`, which is already published on npm, so `release.yml`'s idempotent `npm view` guard would skip the publish and this would never actually ship. Needs a minor bump to `1.11.0` plus a CHANGELOG entry (per the repo's flow both land on `main` with the PR). Two doc nits ride along: the intro paragraph still says the fix is "non-destructive, conforming lines left untouched", which is no longer true for the width fixer (it fully re-lays-out the template, its own comment says so), and `meta.docs.description` still reads "Warn when a class string exceeds the configured print width", which undersells what it now does.

Rebase onto `main` is optional. I checked with a three-way merge and a normal merge does not revert the oxlint 1.80 / type-aware work, it just means pre-merge CI runs on 1.78 instead of 1.80.

Everything else (staircase indent on multi-interpolation templates, tab-width counting, a dead branch, the v4 project-prefix grouping, and the missing edge-case tests) is inline and non-blocking. Happy to pair on the opt-in shape if you want, that's really the only design decision left here.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts [hunk 9]
```diff
@@ -22,6 +23,54 @@ function chunkList(classes: string[], n: number): string[] {
...
+ * Split `classes` into runs of consecutive classes sharing the same variant
+ * chain ("hover:", "md:hover:", "" for base utilities), so wrapping breaks
+ * between variant groups rather than mid-group.
+ */
+function groupByVariantRun(classes: string[]): string[][] {
+  const groups: string[][] = []
+  let current: string[] = []
+  let currentVariant: string | null = null
+  for (const cls of classes) {
+    const variant = getVariantPrefix(cls)
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: nit tied to a documented invariant: `getVariantPrefix` is prefix-unaware, so under a Tailwind v4 project prefix (`@import "tailwindcss" prefix(tw)`) it returns `tw:` for a base utility like `tw:flex` and treats the prefix as a variant. If an unprefixed component class is interspersed, the flanking `tw:` blocks get split into separate runs where the unprefixed equivalent would stay one greedy run. Cosmetic and it converges, but `consistent-variant-order` and `enforce-sort-order` strip the prefix before grouping (see CLAUDE.md's prefix invariant) and this fixer doesn't. No prefix tests here either.

File: packages/oxlint-tailwindcss/tests/rules/enforce-consistent-line-wrapping.test.ts [hunk 10]
```diff
@@ -154,3 +160,97 @@ ruleTester.run(
     ],
   },
 )
+
+// Width-based fixer (printWidth without classesPerLine): classes grouped
+// into runs by variant chain (each run starts its own line), packed to the
+// print width.
+ruleTester.run(
+  'enforce-consistent-line-wrapping (width fixer + variant groups)',
```
Comment (sergioazoc) [inline packages/oxlint-tailwindcss/tests/rules/enforce-consistent-line-wrapping.test.ts:252]: Nice coverage on the base/hover/md:hover groups and nested JSX. To pin the design constraints down, I'd add cases for: important `!` (`!flex`, `items-center!`, `hover:!bg-red-500`), a Tailwind project prefix (`tw:flex tw:hover:...`), tab indentation, a quasi with `${}` on *both* sides (both reserves = 1), and an empty / whitespace-only template (the `classes.length === 0` early return). I checked all five behave correctly today and are idempotent, but none are locked down, so a future change to `getVariantPrefix` or the reserve math would slip through CI.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 1 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: Agreed. I've added a new option `wrapLines` that turns on the new fixer if it is set, and it has the option to either only wrap lines that exceed the printWidth (keeping hand-wrapped templates as-is except for the individual line that exceeded the width) or all lines in the template (as better-tailwindcss's wrapping applied). Option is undefined by default, so existing configs do not have the wrapping applied, and people can opt in to either individual line wrapping or full template wrapping as desired.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 2 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: Updated, with a description of what a leading/hanging quasi is and tests for the various wrapping possibilities.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 3 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: I've added the `group` option from better-tailwindcss, which is used to modify how `wrapLines: all` applies the variant groupings. If `wrapLines: overWidth` is set, groups are ignored and only the individual line that exceeds the printWidth is reflown, preserving hand-formatted layouts for the other lines. This allows either full migration from better-tailwindcss for all group options, or preserving hand-formatting with only tooLong lines being modified.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 4 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: Updated, with tests.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 6 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts:365]: It was intentional, since it represents an issue that cannot be fixed either manually or automatically. Having a warning that can't be actioned waters down/hides actionable warnings, and if a project has a few of these usually results in people ignoring all of that warning type on the assumption that it's just another long class name/they won't be able to do anything about it (report fatigue).

I've added the behaviour change for lines exceeding printWidth containing a single class no longer being reported to the changelog & PR description.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 7 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts:83]: I've added the comment for this in the PR.

A future consideration could be to add a plugin setting for tab width that rules can use as appropriate when calculating things like whether a line exceeds the print width.

File: packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts (hunk 9 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/src/rules/enforce-consistent-line-wrapping.ts]: This moves this rule from DS-independent to DS-optional in order to retrieve the prefix from the Tailwind v4 css file. I've implemented this so that the design system is only consulted when using the variant grouping (wrapLines: all + group: newLine/emptyLine), and it falls back to treating a prefix as part of the variant if the entryPoint was not defined.

File: packages/oxlint-tailwindcss/tests/rules/enforce-consistent-line-wrapping.test.ts (hunk 10 above)
Comment (dawnmist) [reply packages/oxlint-tailwindcss/tests/rules/enforce-consistent-line-wrapping.test.ts:252]: Tests added. :)
